docker/install-dynamic-plugins.py

```diff
@@ -230,7 +230,7 @@ def main():
 
         archive = os.path.join(dynamicPluginsRoot, completed.stdout.decode('utf-8').strip())
 
-        if not package_is_local:
+        if not package_is_local and not skipIntegrityCheck:
```

Comment:
Isn't this additional check redundant since it's not possible to get past this point without `skipIntegrityCheck` not being `True` when the package isn't local?